[2023-06-29T10:31:17.389+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-29T10:31:15.141892+00:00/task_id=create_file_format permission to 509
[2023-06-29T10:31:17.390+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-29T10:31:15.141892+00:00/task_id=create_file_format permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-29T10:31:15.141892+00:00/task_id=create_file_format'
[2023-06-29T10:31:17.443+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-29T10:31:15.141892+00:00/task_id=create_file_format permission to 509
[2023-06-29T10:31:17.444+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-29T10:31:15.141892+00:00/task_id=create_file_format permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-29T10:31:15.141892+00:00/task_id=create_file_format'
[2023-06-29T10:31:17.750+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 2_Test_Badge_5_Setup.create_file_format manual__2023-06-29T10:31:15.141892+00:00 [queued]>
[2023-06-29T10:31:17.764+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 2_Test_Badge_5_Setup.create_file_format manual__2023-06-29T10:31:15.141892+00:00 [queued]>
[2023-06-29T10:31:17.765+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-06-29T10:31:17.786+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): create_file_format> on 2023-06-29 10:31:15.141892+00:00
[2023-06-29T10:31:17.795+0000] {standard_task_runner.py:57} INFO - Started process 1899 to run task
[2023-06-29T10:31:17.800+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', '2_Test_Badge_5_Setup', 'create_file_format', 'manual__2023-06-29T10:31:15.141892+00:00', '--job-id', '665', '--raw', '--subdir', 'DAGS_FOLDER/test_setup.py', '--cfg-path', '/tmp/tmph3pw7jro']
[2023-06-29T10:31:17.803+0000] {standard_task_runner.py:85} INFO - Job 665: Subtask create_file_format
[2023-06-29T10:31:17.873+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-29T10:31:15.141892+00:00/task_id=create_file_format permission to 509
[2023-06-29T10:31:17.874+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-29T10:31:15.141892+00:00/task_id=create_file_format permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-29T10:31:15.141892+00:00/task_id=create_file_format'
[2023-06-29T10:31:17.878+0000] {task_command.py:410} INFO - Running <TaskInstance: 2_Test_Badge_5_Setup.create_file_format manual__2023-06-29T10:31:15.141892+00:00 [running]> on host 4f3859432e6c
[2023-06-29T10:31:18.058+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='2_Test_Badge_5_Setup' AIRFLOW_CTX_TASK_ID='create_file_format' AIRFLOW_CTX_EXECUTION_DATE='2023-06-29T10:31:15.141892+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-06-29T10:31:15.141892+00:00'
[2023-06-29T10:31:18.061+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-06-29T10:31:18.062+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'cd /dbt && dbt run-operation create_file_format --args \'{                            "file_format_name": "ff_json_logs",                             "file_format_type": "JSON",                             "strip_outer_array": true,                             "dry_run": false}\'']
[2023-06-29T10:31:18.078+0000] {subprocess.py:86} INFO - Output:
[2023-06-29T10:31:24.262+0000] {subprocess.py:93} INFO - [0m10:31:24  Running with dbt=1.4.6
[2023-06-29T10:31:27.274+0000] {subprocess.py:93} INFO - [0m10:31:27  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2023-06-29T10:31:27.275+0000] {subprocess.py:93} INFO - There are 1 unused configuration paths:
[2023-06-29T10:31:27.276+0000] {subprocess.py:93} INFO - - models.badge_5.time_of_day_lu
[2023-06-29T10:31:29.403+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-06-29T10:31:29.445+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=2_Test_Badge_5_Setup, task_id=create_file_format, execution_date=20230629T103115, start_date=20230629T103117, end_date=20230629T103129
[2023-06-29T10:31:29.508+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-06-29T10:31:29.660+0000] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
